packages/gamut-labs/src/experimental/PageSection/__tests__/PageSection-test.tsx

```diff
@@ -0,0 +1,71 @@
+import { Text } from '@codecademy/gamut';
+import { setupRtl } from '@codecademy/gamut-tests';
+import { fireEvent, render, screen } from '@testing-library/react';
+import React from 'react';
+
+import { PageSection, PageSectionProps } from '..';
+
+const renderPageSection = (props: Partial<PageSectionProps>) =>
+  setupRtl(PageSection, { title: 'Default Section Title', ...props })();
```

Comment:
This code is trying very hard to recreate the exact functionality of the library already 😛 

```suggestion
import { PageSection } from '..';

const renderView = setupRtl(PageSection, {
  title: 'Default Section Title',
});
```

Reply:
Also, you don't want to use `render(...)` -> `screen...` on their own. Soon components will all use theming that breaks without the test provider set up in `@codecademy/gamut-tests`.

Reply:
Roger that. I'll stick to using `setupRtl`. I think I was thrown off by wanting to nest stuff inside the body of my elements and it didn't occur to me to just pass in `children` directly in the props object of `setupRtl`.

Question though: should I avoid `screen` altogether and always make my queries directly on the rendered view?

Reply:
Yeah that's the style we're going for. In theory `screen` may work but no guarantees it'll stick to that.